<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISourceGen</class>
 <widget class="QDialog" name="UISourceGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SourceGen Alpha 2 (02/25/2017)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox
{
font-size:14px;
}
QLabel
{
font-size:14px;
}
#m_btnExit
{
}

#m_btnExit::hover
{
	color:black;
    border:2px solid #ffaaaa; 
	background:#ffaaaa;
}

#m_btnSave
{
	width:128px;
	font-size:18px;
	padding:8px;
    border-width:3px;
}

#m_btnSave::hover
{
	color:black;
	background:#bbffbb;
	border:3px solid #bbffbb;
}


QDialog
{
	background:#111;
	color:#eefaab;
}

QPushButton
{
	padding:8px;
	border:2px solid #eefaab;
	color:#eefaab;
	border-radius:4px;
    width:96px;
    background:#77000000;
	font-size:14px;
}

QPushButton::hover
{
	background: #eefaab;
	color:black;
}

QLineEdit,QListWidget,QPlainTextEdit
{
	color:#eefaab;
	border:1px solid #eefaab;
	border-radius:4px;
	font-size:18px;
	padding:2px;
background:#77000000;
}

QLineEdit::disabled,QListWidget::disabled,QPlainTextEdit::disabled
{
   background:#77000000;
	color:#333;
	border:1px solid #333;
	border-radius:4px;
	font-size:18px;
	padding:2px;
}

QListWidget::item:selected
{
	background:#eefaab;
	color:black;
	border-radius:4px;
	padding:4px;
	height:28px;
}

QCheckBox
{
	background:transparent;
	color:#eefaab;
	padding:2px;
}

QCheckBox::indicator
{
	color:#eefaab;
}

QLabel
{
	color:#eefaab;
}

QCheckBox::label
{
	color:#eefaab;
}



 QScrollBar:vertical 
{
     border: 2px solid black;
     background:  #77222222;
     width: 15px;
     margin: 22px 0 22px 0;
 }

 QScrollBar::handle:vertical 
{
     background: #eefaab;
     min-height: 20px;
 }

 QScrollBar::add-line:vertical
 {
      border: 2px solid black;
     background:  #77222222;
     height: 20px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

QScrollBar::sub-line:vertical 
{
     border: 2px solid black;
     background:  #77222222;
     height: 20px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical 
{
      border: 2px solid black;
     width: 3px;
     height: 3px;
     background: #eefaab;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{
     background: none;
 }

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>9</number>
       </property>
       <item row="5" column="1">
        <widget class="QLineEdit" name="m_txtNamespace"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="m_txtInherits"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="m_txtClass"/>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>9</number>
         </property>
         <item>
          <widget class="QLineEdit" name="m_txtOutputDir"/>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnBrowse">
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Class Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="m_txtIncludes"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Gets/Sets:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Output Dir:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>9</number>
         </property>
         <item>
          <widget class="QLineEdit" name="m_txtCopyright"/>
         </item>
         <item>
          <widget class="QPushButton" name="m_btnBrowseCopyright">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Class Type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Comma seperated list of include directives to go in the header file.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Format: &amp;lt;file.h&amp;gt; or &amp;quot;file.h&amp;quot;&lt;/p&gt;&lt;p&gt;Example: &amp;lt;file.h&amp;gt;, &amp;quot;file.h&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Includes:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPlainTextEdit" name="m_txtGetSet"/>
       </item>
       <item row="1" column="1">
        <widget class="QListWidget" name="m_lstTypes">
         <property name="currentRow">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Normal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Static</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Singleton</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Virtual</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>QWidget</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>QDialog</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>QMainWindow</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Namespace:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Base Classes:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Copyright File:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="m_btnAbout">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>About</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnHelp">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnClone">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Clone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnClear">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnOutputLog">
         <property name="text">
          <string>View Output Log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_chkDisableGraphics">
         <property name="text">
          <string>Toggle Graphics</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_chkOverwrite">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Overwrite</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_chkSaveToSubfolder">
         <property name="text">
          <string>Save to Subfolder:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="m_frmSubFolder">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item row="2" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Source:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="m_cmbHeaderFolder">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="m_cmbUIFolder">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>UI:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="m_cmbSourceFolder">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Header:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnSave">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnExit">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_lstTypes</tabstop>
  <tabstop>m_txtClass</tabstop>
  <tabstop>m_txtIncludes</tabstop>
  <tabstop>m_txtInherits</tabstop>
  <tabstop>m_txtNamespace</tabstop>
  <tabstop>m_txtOutputDir</tabstop>
  <tabstop>m_btnBrowse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
